_includes/deep_learning_content.html

```diff
@@ -0,0 +1,81 @@
+{% if page.deep-learning-thank-you == true %}
+<body class="deep-learning-thank-you">
+    
+    <div class="container-fluid header-holder">
```

Comment:
Double-check your indentation in this file. You should stick to using 2 spaces since that's what the rest of the project uses.